src/legacy/core_plugins/kibana/public/management/sections/index_patterns/_index.scss

```diff
@@ -0,0 +1,9 @@
+@import 'variables';
+@import 'mixins';
+
+@import 'hacks';
+
+@import 'assets/index';
+@import 'create_index_pattern_wizard/index';
+@import 'empty_index_pattern_prompt/index';
+@import 'index_pattern_table/index';
```

Comment:
This stuff can probably be imported into the individual components, and then use the a more broad import individually into those files for the vars/mixins as needed.

Reply:
This section isn't in the NP yet, so we have to stick with our old SASS manifests.